src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -521,7 +522,7 @@ public function add(string $body, array $headers, int $delayInMs = 0): string
 
         try {
             if ($delayInMs > 0) { // the delay is <= 0 for queued messages
-                $id = bin2hex(random_bytes(4));
+                $id = (string) Uuid::v7();
```

Comment:
Would it make sense to inject a UUID factory into this connection?

Shall we use the base58 representation to get more compact identifiers?

Reply:
Using the base58 is a good idea. Using a factory IMO complicates things too much for the gain it brings.